src/lib/containers/GoogleMap/SynapseUserMarker.tsx

```diff
@@ -0,0 +1,75 @@
+import { Divider } from '@material-ui/core'
+import { InfoWindow, Marker } from '@react-google-maps/api'
+import React, { useMemo } from 'react'
+import { SMALL_USER_CARD } from '../../utils/SynapseConstants'
+import Typography from '../../utils/typography/Typography'
+import UserCard from '../UserCard'
+import { GeoData } from './GeoData'
+
+const MAP_MARKER_IMAGE_URL =
+  'https://s3.amazonaws.com/static.synapse.org/images/synapse-map-marker.png'
```

Comment:
I imagine this will need to be configurable for portals, but that can come later